src/pybind/mgr/osd_perf_query/module.py

```diff
@@ -3,14 +3,27 @@
 osd_perf_query module
 """
 
+from time import time
+import errno
+import prettytable
+
 from mgr_module import MgrModule
 
+def get_human_readable(bytes, precision=2):
```

Comment:
Nit: we have switched Ceph to IEC units (i.e. 1 MiB -> 1024 KiBs) [1]

[1] https://github.com/ceph/ceph/blob/master/src/include/types.h#L444

Reply:
Do you want me just to fix the base (1024), or do you also want I add 'i' to the suffixes ('Ki', 'Mi', ...)?

Reply:
Preferably both to avoid confusion.

Reply:
I wanted to avoid 'B' in the suffix name, because I already have it in the column name ('read_bytes/sec'), and just 'K', 'M',... suffixes looked ok to me (similarly to how we can specify image size, io-size, etc in rbd commands -- 4K, 1M,...). I don't recall seeing 'Ki', 'Mi' (without 'B').

Reply:
It's the correct IEC prefix [1] to avoid confusion with SI prefixes.

[1] https://en.wikipedia.org/wiki/Binary_prefix

Reply:
@dillaman Thanks. updated